content/en/docs/concepts/security/pod-security-standards.md

```diff
@@ -70,7 +70,9 @@ enforced/disallowed:
 				spec.hostNetwork<br>
 				spec.hostPID<br>
 				spec.hostIPC<br>
-				<br><b>Allowed Values:</b> false<br>
+				<br><b>Allowed Values:</b><br>
+        false<br>
+        undefined / nil<br>
```

Comment:
suggest `undefined / false`... nil is not actually permitted to be sent for this field